.coveragerc

```diff
@@ -0,0 +1,5 @@
+# .coveragerc to control coverage.py
+[run]
+
+[report]
+include = */site-packages/google/datalab/*
```

Comment:
I don't think this works, `include` is used to filter the report with the provided pattern, it doesn't actually format the file paths.